support-frontend/app/controllers/UserDigitalSubscription.scala

```diff
@@ -1,37 +0,0 @@
-package controllers
-
-import actions.CustomActionBuilders.AuthRequest
-import com.gu.monitoring.SafeLogger
-import play.api.mvc.Result
-import play.api.mvc.Results._
-import play.mvc.Http.Status.FOUND
-import services.{AccessCredentials, AuthenticatedIdUser, MembersDataService}
-import SafeLogger._
-import scala.concurrent.{ExecutionContext, Future}
-
-object UserDigitalSubscription {
-  def userHasDigitalSubscription(
```

Comment:
we don't actually use this code any more, I think this was to stop duplicate purchases